features/availability.feature

```diff
@@ -62,6 +62,7 @@ Feature: Availability reports
 		And I should see "Report Settings"
 
 	@reports
+	@unreliable
```

Comment:
Skipping a problematic scenario is OK by me.
Might want to rename it unreliable_el8 or something to that effect.

Reply:
I think it might not be necessary since I did other changes to the test. Will try without it.